### ISSUE TITLE
apps pages are blank with 17.0.2

### ISSUE THREAD

**Issue by jpimlott** [nextcloud/vm#1019]
## Steps to reproduce
1. install nc from techandme with virtual machine
2.update php with missing part mentioned in the security check.
3.go to apps page and apps that are supplied with nc are there but non of the pages to add more apps are available

## Expected behaviour
Tell us what should happen
apps on pages for different ones like security, files...  should ones listed to install
## Actual behaviour
Tell us what happens instead
all of the pages are blank.
## Server configuration

**Nextcloud server version:** (see your admin page)
17.0.2
**Server OS** (Ubuntu server is default)
ubunto 
**How did you install the VM?** (Scripted install from master **OR** Released version)
scripted install from techandme
## Network
**Do you use DHCP?**
no , hardcoded 
**Is port 80 and/or 443 open?**
yes 
#### Log file (/var/log/nextcloud/nextcloud.log)
```
Insert your nextcloud.log file here
```

#### Installation log (command output)
```
Insert the CLI output here
```

**Comment by enoch85** [comment]
Thanks for posting your issue!

> 2.update php with missing part mentioned in the security check.

Do you mean imagick? Maybe you installed the wrong package? Imnagick is not recomended due to security reasons. That's why we left it out in the VM.

**Comment by enoch85** [comment]
I just tested a fresh install on current master and everything works. ---> closing.

**Comment by jpimlott** [comment]
The problem comes and goes.  Holding shift then pressing the refresh button has helped
It still does it some of the time


The mail app does not show mail in the inbox sometimes either

John


Sent from my iPhone

On Jan 12, 2020, at 11:18, Daniel Hansson <notifications@github.com> wrote:

﻿


Closed #1019<https://github.com/nextcloud/vm/issues/1019>.

**Comment by enoch85** [comment]
@jpimlott Check your internet connection. The App Store fetches live data, so my guess something is up with your ISP.
